flutter_searchbox_ui/lib/src/utils.dart

```diff
@@ -66,3 +67,59 @@ Location? getLocationObject(dynamic location) {
     }
   }
 }
+
+List prepareValueList(Map value) {
+  if (value.isEmpty) {
+    return [];
+  }
+  List valueList = [];
+  if (value.containsKey('start')) {
+    valueList.add(value['start']);
+  }
+  if (value.containsKey('end')) {
+    valueList.add(value['end']);
+  }
+  return valueList;
+}
+
+String processFilterValues(dynamic value) {
+  if (value == null || value.isEmpty) {
+    return "";
+  }
+  if (value is String) {
+    return value;
+  } else if (value is num) {
+    return value.toString();
+  } else if (value is List) {
+    return value.join(", ");
+  } else if (value is Map) {
+    return processFilterValues(prepareValueList(value));
+  } else {
+    return value.toString();
```

Comment:
@mohdashraf010897 Do you handle empty string? It shouldn't display empty string as a selected filter.

Reply:
yes 
` if (value == null || value.isEmpty) {
    return "";
  }`